sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -666,11 +702,116 @@ const StartHTLCAssetLockListener = (
     assetId: string,
     recipientECert: string,
     lockerECert: string,
-    lockCallback: (c: Contract, d: string, t: string, i: string, r: string, l: string, v: string) => any,
+    lockCallback: (c: Contract, d: string, t: string, i: string, r: string, l: string, v: Hash, timeout: number) => any,
 ): void => {
     StartHTLCEventListener(contract, 'LockAsset', contractId, assetType, assetId, -1, recipientECert, lockerECert, lockCallback);
 }
 
+// NOTE: For Nonfungible Assets
+// Byzantine Swaps: Timed counterpart for timed AssetLockListener for reversion
+const assetLockExpirationCallback = (
+    contract: Contract, 
+    contractID: string, 
+    assetType: string, 
+    assetID: string, 
+    recipientECert: string, 
+    lockerECert: string, 
+    hash: Hash,
+    expiryTime: number
+): void => {
+    // Compare expiryTimeSec with currentTimeSec, which is number of milliseconds since epoch (L174)
+    const currTimeSecs = Math.floor(Date.now()/1000);
+    const reclaimCallback = async (contract: Contract, contractID: string) => {
+        // Check if asset hasn't been claimed yet. If true, do nothing. If false, either do the following cases:
+        const [islocked, isLockedQueryError] = await helpers.handlePromise(isAssetLockedInHTLCqueryUsingContractId(contract, contractID));
+        if (islocked == false) {
```

Comment:
Shouldn't there be quotes around `false`? And a `===`?

Reply:
No quotes required, because in SDK we made the change to return boolean. But yes `===` should be used.